spec/support/database.rb

```diff
@@ -1,13 +1,45 @@
 # frozen_string_literal: true
 
-module TemporalTables
-  module DatabaseAdapter
-    def self.adapter_name
```

Comment:
Not sure if this namespace is really needed or not, I removed it while refactoring (it wasn't used anywhere afaik).